pkg/sql/schemachanger/scbuild/internal/scbuildstmt/alter_table_add_column.go

```diff
@@ -69,7 +69,9 @@ func alterTableAddColumn(
 					"column name %q conflicts with a system column name",
 					d.Name))
 			}
-			panic(sqlerrors.NewColumnAlreadyExistsInRelationError(string(d.Name), tn.Object()))
+			if colTargetStatus != scpb.ToAbsent {
```

Comment:
and the code change @rafiss 

(unless i worded something incorrectly for the release note)